cedar-policy-core/src/est/err.rs

```diff
@@ -80,6 +80,10 @@ pub enum FromJsonError {
     #[error("invalid entity type: {0}")]
     #[diagnostic(transparent)]
     InvalidEntityType(ParseErrors),
+    /// Errors constructing AST expressions
+    #[error(transparent)]
+    #[diagnostic(transparent)]
+    ExprConstructionError(#[from] ast::ExprConstructionError),
```

Comment:
What's this doing? Feels orthogonal to the rest of the PR.

Reply:
Removed.